lib/pure/options.nim

```diff
@@ -139,6 +139,13 @@ proc get*[T](self: Option[T], otherwise: T): T =
   else:
     otherwise
 
+proc mget*[T](self: var Option[T]): var T =
+  ## Returns contents of the Option. If it is none, then an exception is
+  ## thrown.
+  if self.isNone:
+    raise UnpackError(msg : "Can't obtain a value from a `none`")
```

Comment:
Bugs should not be mapped to exceptions. This should use ``assert``/``doAssert``.

Reply:
Agreed!

Reply:
This is consistent with the rest of ``options``. We can fix it in a separate PR.

Reply:
Fair enough.

Reply:
Nitpick: No space before a colon.